addons/common/functions/fnc_dumpPerformanceCounters.sqf

```diff
@@ -0,0 +1,56 @@
+/*
+ * Author: ACE-Team, ZEN Team
+ * Dumps performance counter statistics into RPT.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call zen_common_fnc_dumpPerformanceCounters
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+diag_log text "~~~ ZEN PERFORMANCE COUNTERS DUMP ~~~";
```

Comment:
Couldn't you use the CBA macro `LOG`? And `TRACE_x` for all the others that have data?

Reply:
That would add the formatting that those macros come with. Output is cleaner like this.

Reply:
But you have `diag_logs` couple of lines down that use formats. It's cleaner to use the macros.

Reply:
Plus, is there any reason that `text` is called? Doesn't look very structured text to me.

Reply:
diag_log "asdf" => `"asdf"` in file
diag_log text "asdf" => `asdf` in file

Reply:
Ah, I see.